### DISCUSSION TITLE
Hardware test on v2 PCB

### DISCUSSION THREAD

**Discussion by chumleychuff** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#592] [Q&A] [answered]
Sorry if this is a dumb question but how to you do the hardware test on a v2 PCB? 

I can't see the same pins as the v1 image on the hardware test page.

**Comment by visualapproach** [comment]
The picture is just for reference. It works the same as long as you have configured correct pcb. Just connect the cables to each other and provide power.

**Reply by chumleychuff** [reply]
Thanks for the prompt response - I have connected a 5v power to pins H1B and it powers the board however I cannot pass the hardware test - I assumed it was my cable / soldering so tried a different board with just the ESP8266 and

attached breadboard cables to the right pins D1 to D6 etc. 
checked continuity with my multimeter - which confirmed D1 to D6 , D2 to D4, D5 to D3 and NC to D7.
ensured the hardware config page has the right pin out 
but the hardware test still fails :(

could I have uploaded the software incorrectly? or is the hardware checker not working correctly maybe?

**Reply by visualapproach** [reply]
image on your setup?

**Comment by chumleychuff** [comment]
ESP8266 is running build from Master (v4) from Saturday
I have tried v2 and v2b boards - pics of my v2b board - excuse the tape it was just helping to keep the pins in place when soldering #embarrassed_face

![1](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/54021695/ca1b141a-9c80-4431-a44d-4ae1c5c1de91)

![2](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/54021695/3abab451-4166-4ca1-8203-cdcb01c3cbba)

![3](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/54021695/0a35cec5-a2f4-4c23-b618-ab8ad0f89072)

**Reply by visualapproach** [reply]
The level converter is missing...

**Reply by chumleychuff** [reply]
would that fail the hardware test? I just wanted to ensure the pins are correct before putting all the other stuff on.

**Reply by visualapproach** [reply, marked as answer]
Certainly! No connection from esp to wires

**Reply by chumleychuff** [reply]
haha i'm so dumb - thank you!

Coffee ☕ sent!

**Reply by visualapproach** [reply]
Thank you @chumleychuff 🤗
